iOS/G3MiOSSDK/Commons/Basic/Tile.cpp

```diff
@@ -648,7 +653,9 @@ ElevationData* Tile::createElevationDataSubviewFromAncestor(Tile* ancestor) cons
   if ((_lastElevationDataProvider != NULL) &&
       (_lastTileMeshResolutionX > 0) &&
       (_lastTileMeshResolutionY > 0)) {
-    return new DecimatedSubviewElevationData(ed,
+      
+      //return ed;
+    return new InterpolatedSubviewElevationData(ed,
                                              _sector,
                                              Vector2I(_lastTileMeshResolutionX, _lastTileMeshResolutionY));
   }
```

Comment:
Why did it change from DecimatedSubviewElevationData to InterpolatedSubviewElevationData?